src/js/BookReader.js

```diff
@@ -302,12 +302,38 @@ BookReader.prototype.initParams = function() {
     }
   }
 
+  /*
+  // If we have a querystring
+  params.queryString = new URLSearchParams(this.readQueryString());
+
+  // Check for Search plugin and term
+  if (this.options.enableSearch) {
+    this.options.initialSearchTerm =
+      this.options.initialSearchTerm ??
+      params.queryString.get('q') ??
+      params.search;
+  }
+*/
   // Set for init process, return to false at end of init()
   this.suppressFragmentChange = !params.fragmentChange;
 
   return params;
 }
 
+/**
+ * Return URL or fragment querystring
+ */
+/*
+BookReader.prototype.readQueryString = function() {
```

Comment:
same question here - is this meant to be runnable?

Reply:
As above